rotkehlchen/chain/ethereum/modules/thegraph/constants.py

```diff
@@ -1,12 +1,11 @@
 from typing import Final
 from rotkehlchen.chain.evm.decoding.types import CounterpartyDetails
+from rotkehlchen.chain.evm.types import string_to_evm_address
 
 CPT_THEGRAPH: Final = 'thegraph'
-
 THEGRAPH_CPT_DETAILS = CounterpartyDetails(
     identifier=CPT_THEGRAPH,
     label='The Graph',
     image='thegraph.svg',
 )
```

Comment:
I didn't notice before but the counterpart is duplicated. You can remove it from here and use the constatn in evm

Reply:
same for the `CPT_THEGRAPH` string